refresh.template.py

```diff
@@ -227,6 +239,28 @@ def _get_headers(compile_args: typing.List[str], source_path: str):
 _get_headers.has_logged_missing_file_error = False
 
 
+def _params_file_to_source_files(compile_args: typing.List[str]):
```

Comment:
This and the changes to get_files are no longer needed, right?

Reply:
Sure, will remove